test/x3/linker_error_test/ast.hpp

```diff
@@ -0,0 +1,76 @@
+//
+// Created by lukas on 11.11.16.
+//
+
```

Comment:
Odd comment. Please be consistent with the rest of the codebase. Who is lukas?

Reply:
I don't know a lukas.  I assume it was just copy&pasted from the original problem code
mentioned in Purpose: at top of main.cpp.

Will remove.

Reply:
How about simply add these tests in one of the existing tests. Perhaps rule_tests?

Reply:
It's only 1 test and it seemed easier to just use  a glob to select all the .cpp's for that test.
Separate .cpp's were used because that was the original author's design and it conforms
to what I thought was the accepted practice for using BOOST_SPIRIT_INSTANTIATE.
I guess I could just combine all the .cpp's into one file.  Is that what you mean?